packages/samples/headless-ssr-commerce/next.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  // Enable compiling of common components between app and pages router outside `src/`.
+  transpilePackages: ['@coveo/headless-ssr-samples-common'],
+  reactStrictMode: true,
+};
+
+module.exports = nextConfig;
```

Comment:
I just saw this as well. We won't be using headless-ssr-samples-common in this project. I am pretty sure that was the common folder for the page-router/app-router split.  How did you create the sample ? Did you create a new nextjs project or copied the old one we had ?